Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/profileSettings/profileProperties/profilePropertyRow/index.jsx

```diff
@@ -14,7 +13,7 @@ class ProfilePropertyRow extends Component {
 
     toggle() {
         if ((this.props.openId !== "" && this.props.id === this.props.openId)) {
-            //this.props.Collapse();
```

Comment:
This Collapse is not the component but a function passed down as props, this one should not get renamed to Collapsible